base_layer/wallet/src/testnet_utils.rs

```diff
@@ -307,6 +326,14 @@ pub fn generate_wallet_test_data<
 
     thread::sleep(Duration::from_millis(1000));
 
+    let _ = wallet_alice.shutdown();
+    let _ = wallet_bob.shutdown();
+
+    let _ = std::fs::remove_dir_all(&alice_temp_dir).unwrap_or_default();
```

Comment:
If you want to squash the error this is how you do it
```suggestion
    let _ = std::fs::remove_dir_all(&alice_temp_dir);
```

Reply:
thanks, done